### PR TITLE
Create article

### PR DESCRIPTION
**Pull request by Lex5mith**
Added create article page 
Needs work on validation

### CODE REVIEW

File: src/components/CreateComment.jsx [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+      postComment(article_id_from_params, user.username, commentText)
+        .then((data) => {
+          setShowCommentInput(!showCommentInput);
+          setIsLoading(false);
+          setCommentText("");
+          console.log(data);
+        })
+        .catch((error) => {
+          setIsLoading(false);
+          setIsError(error);
```
Comment (johnhutcheon) [inline src/components/CreateComment.jsx:54]: This all looks good where you're setting the error, but at the moment the comment posts then when we refresh it isn't there. I think this is fine to have it optimistically rendering, but can we get some sort of message to the user like "Something went wrong" ?

**Review by johnhutcheon — CHANGES_REQUESTED** [review]
hey Lex. You've actually done post a new article here which wasn't the ticket, so I've just commented on the post a new comment code (although your post new article stuff all looks good too!).
If you just see my comment, could we get some sort of user feedback for if a comment has not posted? Great work, your styling is amazing! 🙌

File: src/components/CreateComment.jsx (hunk 1 above)
Comment (Lex5mith) [reply src/components/CreateComment.jsx:54]: yep, have this on a list that i'm keeping for dealing with error handling when I've finished building everything. 🧘‍♀️

Comment (Lex5mith) [reply src/components/CreateComment.jsx:54]: also MUI makes styling WAAAAYYYYY easier than trying to wrap my brain around css 😂

**Pull request merged**